scripts/aws/find_supported_instance_types.rb

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# This script outputs all instance types that satisfy the given AMI and Amazon Availability Zone
+
+require 'aws-sdk-core'
+require 'aws-sdk-ec2'
+require 'optparse'
+require 'yaml'
+
+MDBCI_CONFIG_FILE_PATH = '~/.config/mdbci/config.yaml'
```

Comment:
Use ConfigurationReader module to get path to the `config.yaml` file. This way you won't need to hard-code path.